tools/test_check_consistency.py

```
@@ -0,0 +1,371 @@
+                "metadata": {"alternative": "test subject"},
+            },
+            {
+                "id": "animal-violence.livestock",
+                "pattern-regex": "\\blivestock\\b",
+                "languages": ["generic"],
+                "severity": "WARNING",
+                "metadata": {"alternative": "farmed animals"},
+            },
+        ])
+        findings = check_semgrep(canonical, repos_dir)
+        self.assertEqual(len(findings), 0)
```

Ruff (0.15.7)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)